### PR TITLE
Use `upi finish` subcommand

### PR DESCRIPTION
**Pull request by vrutkovs**
This would use kni-installer's subcommand which waits for all operators to rollout.

Requires #262 to make image-registry rollout properly

### CODE REVIEW

File: utils.sh [hunk 1]
```diff
@@ -30,6 +30,13 @@ function create_cluster() {
     $GOPATH/src/github.com/openshift-metalkube/kni-installer/bin/kni-install --dir "${assets_dir}" --log-level=debug create cluster
 }
 
+function wait_for_cvo_finish() {
+    local assets_dir
+
+    assets_dir="$1"
+    $GOPATH/src/github.com/openshift-metalkube/kni-installer/bin/kni-install --dir "${assets_dir}" --log-level=debug upi finish
```
Comment (stbenjam) [inline utils.sh]: Doesn't this require that all operators come up? Are we there yet? I thought the last successful run I had not everything came up

Comment (vrutkovs) [reply utils.sh]: Yes, it would. In my tests the only misbehaving operator was image-registry - and #262 would fix that

File: utils.sh [hunk 2]
```diff
@@ -146,13 +153,6 @@ function master_node_to_install_config() {
 EOF
 }
 
-function collect_info_on_failure() {
-    $SSH -o ConnectionAttempts=500 core@$IP sudo journalctl -b -u bootkube
-    oc get clusterversion/version
-    oc get clusteroperators
-    oc get pods --all-namespaces | grep -v Running | grep -v Completed
-}
```
Comment (stbenjam) [inline utils.sh]: Can you remove the call to this in https://github.com/openshift-metalkube/dev-scripts/blob/master/06_create_cluster.sh#L36? I don't think it should be there

**Comment by stbenjam** [comment]
Otherwise LGTM.

File: utils.sh (hunk 2 above)
Comment (stbenjam) [reply utils.sh]: (Or restore this, if we do want to collect those logs)

Comment (vrutkovs) [reply utils.sh]: Done

Comment (vrutkovs) [reply utils.sh]: This is no longer needed - it was an attempt to collect logs when install fails. It didn't work and got properly implemented in `run-ci`

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/536/

**Comment by stbenjam** [comment]
Looks good, tested locally as well

**Pull request merged**